script.js

```diff
@@ -0,0 +1,41 @@
+const articleContainer = document.querySelector(".article-container");
```

Comment:
to select an element from dom always use ids

Reply:
Okay i'll improve it

Reply:
I have corrected it.